detector_test.go

```diff
@@ -789,89 +1135,67 @@ func hasEntries(a, b []lifecycle.BuildPlanEntry) bool {
 	return true
 }
 
-func allLogs(logHandler *memory.Handler) string {
-	var out string
-	for _, le := range logHandler.Entries {
-		out = out + le.Message + "\n"
-	}
-	return h.CleanEndings(out)
-}
-
-const outputFailureEv1 = `
-======== Output: A@v1 ========
-detect out: A@v1
-detect err: A@v1
-======== Output: C@v1 ========
-detect out: C@v1
-detect err: C@v1
-======== Output: B@v1 ========
-detect out: B@v1
-detect err: B@v1
-======== Results ========
-fail: A@v1
-fail: C@v1
-fail: B@v1
-======== Output: A@v1 ========
-detect out: A@v1
-detect err: A@v1
-======== Output: B@v2 ========
-detect out: B@v2
-detect err: B@v2
-======== Results ========
-fail: A@v1
-fail: B@v2
-======== Output: A@v1 ========
-detect out: A@v1
-detect err: A@v1
-======== Output: C@v2 ========
-detect out: C@v2
-detect err: C@v2
-======== Output: D@v2 ========
-detect out: D@v2
-detect err: D@v2
-======== Output: B@v1 ========
-detect out: B@v1
-detect err: B@v1
-======== Results ========
-fail: A@v1
-fail: C@v2
-fail: D@v2
-fail: B@v1
-======== Output: A@v1 ========
-detect out: A@v1
-detect err: A@v1
-======== Output: B@v1 ========
-detect out: B@v1
-detect err: B@v1
-======== Results ========
-fail: A@v1
-fail: B@v1
-======== Output: A@v1 ========
-detect out: A@v1
-detect err: A@v1
-======== Output: D@v1 ========
-detect out: D@v1
-detect err: D@v1
-======== Output: B@v1 ========
-detect out: B@v1
-detect err: B@v1
-======== Results ========
-fail: A@v1
-fail: D@v1
-fail: B@v1
-`
-
-func tofile(t *testing.T, data string, paths ...string) {
-	t.Helper()
-	for _, p := range paths {
-		f, err := os.OpenFile(p, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0777)
-		if err != nil {
-			t.Fatalf("Error: %s\n", err)
-		}
-		if _, err := f.Write([]byte(data)); err != nil {
-			f.Close()
-			t.Fatalf("Error: %s\n", err)
-		}
-		f.Close()
-	}
-}
+// TODO: see if this can be removed
```

Comment:
TODO

This const was used to test the `it` for `should expand order-containing buildpack IDs`. I think now that the calls to `Resolver` are mocked, we can remove it since it's verifying the same thing (order of resolution).